week-4/F-object-keys/exercise-part-0.js

```diff
@@ -20,13 +20,13 @@ var highScores = {
 
 // ONLY EDIT BELOW HERE
 
-var capitalCitiesKeys = ;
-var highScoresKeys;
+var capitalCitiesKeys = capitalCities;
```

Comment:
I'm afraid it's a little more complex than this... have you executed the code to see if it works?